projects/plugins/jetpack/modules/masterbar/admin-menu/class-p2-admin-menu.php

```diff
@@ -139,7 +143,11 @@ private function remove_menus_for_hub() {
 		// Hide appearance.
 		remove_menu_page( $this->appearance_slug );
 		// Tools.
-		remove_menu_page( $this->tools_slug );
+		remove_submenu_page( $this->tools_slug, 'https://wordpress.com/marketing/tools/' . $this->domain );
+		remove_submenu_page( $this->tools_slug, 'https://wordpress.com/earn/' . $this->domain );
+		remove_submenu_page( $this->tools_slug, 'https://wordpress.com/import/' . $this->domain );
+		remove_submenu_page( $this->tools_slug, 'https://wordpress.com/export/' . $this->domain );
+		add_submenu_page( $this->tools_slug, esc_attr__( 'Integrations', 'jetpack' ), __( 'Integrations', 'jetpack' ), 'manage_options', 'https://wordpress.com/marketing/connections/' . $this->domain, null, 0 );
```

Comment:
```suggestion
		add_submenu_page( $this->tools_slug, __( 'Integrations', 'jetpack' ), __( 'Integrations', 'jetpack' ), 'manage_options', 'https://wordpress.com/marketing/connections/' . $this->domain, null, 0 );
```

Is there a reason behind using `esc_attr__` vs just `__` here?